package.json

```diff
@@ -74,18 +81,28 @@
     "gulp-sourcemaps": "1.6.0",
     "gulp-uglify": "1.5.3",
     "gulp-util": "3.0.7",
+    "handlebars": "4.0.5",
+    "html-loader": "0.4.3",
+    "html-webpack-plugin": "2.16.0",
+    "image-webpack-loader": "1.7.0",
     "jasmine-reporters": "2.1.1",
     "jest-cli": "11.0.2",
     "jest-webpack-alias": "2.2.0",
     "json-loader": "0.5.4",
+    "less": "2.6.1",
+    "less-loader": "2.2.3",
     "localStorage": "1.0.3",
     "marked": "0.3.5",
     "mkdirp": "0.5.1",
     "node-libs-browser": "1.0.0",
+    "postcss-loader": "0.8.2",
     "run-sequence": "1.1.5",
     "source-map-loader": "0.1.5",
+    "string-replace-webpack-plugin": "0.0.3",
+    "style-loader": "0.13.1",
     "transform-loader": "0.2.3",
     "webpack": "1.13.0",
+    "webpack-dev-server": "1.14.1",
```

Comment:
Oh maaan, so many new dependencies :( I hope we're removing some other ones to offset.